### ISSUE TITLE
Unable to see NgModules and Router Tree in  Augury Extention

### ISSUE THREAD

**Issue by vinayns1bm14cs106** [rangle/augury#1475]
Augury: 1.25.2
Date: Sat, 16 May 2020 22:49:13 GMT
OS: Win32

Demo test application:
-- Git repository for demo app showing the issue (optional but very helpful for difficult issues).
-- If a code snippet will completely show the issue, please include it.

Description of issue:
-- Include (clipped) screenshot images if possible.

Angular version (required): 9.1.7

Steps to reproduce:
Not able to see NgModules, Router Tree, Injector Graph and all. 
![augury](https://user-images.githubusercontent.com/28885246/82131732-f3b92400-97f5-11ea-8e80-e1a4d25f8331.PNG)

Additional details:
What is the issue

**Comment by herobarua** [comment]
Yes it not working with latest version for Angular 9.1.7, i hope it will be fix As soon as possible.

it working with good Angular version 8.2.14.

**Comment by AleksanderBodurri** [comment]
Duplicate: https://github.com/rangle/augury/issues/1472
